Packs/Phishing/ReleaseNotes/2_2_8.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### FindDuplicateEmailIncidents
+- Increased timeout so the script be able to run when having large amount of data.
```

Comment:
```suggestion
- Increased the *timeout*of the script in order it will be able to process a large number of incidents.
- Updated the Docker image to: *demisto/sklearn:1.0.0.23593*.
```